sql.d/schema.sql

```diff
@@ -408,6 +409,7 @@ CREATE TABLE links (
 -- Problems
 --------------------------------------------------------------------------------
 CREATE TABLE problems (
+    package_id int NOT NULL,
```

Comment:
Minor nit: the indentation on this line seems to be inconsistent with the ones below (looks like a tabs vs. spaces issue)

Reply:
Thanks! Will fix :)